restapi/admin_tenants.go

```diff
@@ -989,7 +990,7 @@ func getTenantUsageResponse(session *models.Principal, params admin_api.GetTenan
 		log.Println("error getting admin info:", err)
 		return nil, err
 	}
-	info := &models.TenantUsage{UsedSize: adminInfo.Usage}
+	info := &models.TenantUsage{UsedSize: adminInfo.Usage, DiskUsedSize: adminInfo.DisksUsage}
```

Comment:
```suggestion
	info := &models.TenantUsage{Usage: adminInfo.Usage, DiskUage: adminInfo.DisksUsage}
```

DiskUsedSize doesn't make sense

Reply:
sure, done